MekHQ/src/mekhq/campaign/mission/AtBDynamicScenarioFactory.java

```diff
@@ -457,10 +582,34 @@ public static int generateForce(AtBDynamicScenario scenario, AtBContract contrac
 
             if (forceTemplate.getAllowAeroBombs()) {
                 MapLocation mapLocation = scenario.getTemplate().mapParameters.getMapLocation();
-                boolean isAeroMap = (mapLocation == MapLocation.LowAtmosphere) ||
-                        (mapLocation == MapLocation.Space);
+                int ownerBaseQuality;
+                boolean isPirate = faction.isPirate();
+
+                // Use the raw quality values rather than the diluted 'effective' rating
+                switch (forceAlignment) {
+                    case Allied:
+                        ownerBaseQuality = contract.getAllyQuality();
+                        break;
+                    case Opposing:
+                        ownerBaseQuality = contract.getEnemyQuality();
+                        break;
+                    case Third:
+                        // Slight hack, assume "Unidentified Hostiles" are pirates with variable
+                        // quality
+                        ownerBaseQuality = Compute.randomInt(3);
+                        isPirate = forceTemplate.getForceName().toLowerCase().contains("unidentified");
+                        break;
+                    default:
+                        ownerBaseQuality = quality;
+                        break;
+                }
 
-                populateAeroBombs(generatedLance, campaign, !isAeroMap);
+                // Load the fighters with bombs
+                populateAeroBombs(generatedLance,
+                        campaign,
+                        (mapLocation != MapLocation.Space && mapLocation != MapLocation.LowAtmosphere),
```

Comment:
Is there a reason to not use `!isAeroMap` for the third argument here?
The new code will prevent bombers from being loaded in space; space bombing is allowed both in the rules and the game.
If there _is_ a reason to change this check, should it not also be updated in `AtBScenario.java`?

Reply:
NVM, I see that the use of this argument has been changed further on.